larray/core/array.py

```diff
@@ -5211,6 +5215,15 @@ def equals(self, other, nan_equals=False):
         bool
             Returns True if self is equal to other.
 
+        Notes
+        -----
+        For finite values, equals uses the following equation to test whether two floating point values are equivalent.
```

Comment:
this is not only valid for floating point values (and it makes sense to use atol or rtol on int arrays too).

Reply:
s/equivalent/equal/ Otherwise the wording might be ambiguous when we introduce .equiv (#237)